.eslintrc.json

```diff
@@ -0,0 +1,12 @@
+{
+    "extends": "eslint:recommended",
+    "env": {
+        "browser": true,
+        "es6": true,
```

Comment:
depending on what's merged first, we will remove it in either this PR or the other. otherwise it will not be a clean run (it has only 1 failure right now).